#!/bin/sh /usr/share/dpatch/dpatch-run
## 209-magic-update-pcp.dpatch by Werner Fink <werner@suse.de>
##
## DP: Correct PCP entries.

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/sgi file-4.20/magic/Magdir/sgi
--- file-4.20.orig/magic/Magdir/sgi	2005-06-22 21:42:48.000000000 +0000
+++ file-4.20/magic/Magdir/sgi	2007-04-03 14:18:32.000000000 +0000
@@ -8,12 +8,12 @@
 0	string	PmNs				PCP compiled namespace (V.0)
 0	string	PmN				PCP compiled namespace
 >3	string	>\0				(V.%1.1s)
-3	lelong	0x84500526			PCP archive
+3	belong	0x84500526			PCP archive
 >7	byte	x				(V.%d)
->20	lelong	-2				temporal index
->20	lelong	-1				metadata
->20	lelong	0				log volume #0
->20	lelong	>0				log volume #%ld
+>20	belong	-2				temporal index
+>20	belong	-1				metadata
+>20	belong	0				log volume #0
+>20	belong	>0				log volume #%ld
 >24	string	>\0				host: %s
 0	string	PCPFolio			PCP 
 >9	string	Version:			Archive Folio
@@ -27,6 +27,9 @@
 0	string	#pmlogger			PCP pmlogger config
 >10	string	Version
 >18	string	>\0				(V%1.1s)
+0	string	#pmdahotproc			PCP pmdahotproc config
+>13	string	Version
+>21	string	>\0				(V%-3.3s)
 0	string	PcPh				PCP Help 
 >4	string	1				Index
 >4	string	2				Text
